apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/SpannerCommandTests.cs

```diff
@@ -110,67 +111,107 @@ public void CloneWithQueryOptions()
         public void CommandHasConnectionQueryOptions()
         {
             const string connOptimizerVersion = "1";
-            Mock<SpannerClient> spannerClientMock = SetupExecuteStreamingSql(connOptimizerVersion);
+            const string connOptimizerStatisticsPackage = "stats_package_1";
+            Mock<SpannerClient> spannerClientMock = SpannerClientHelpers
+                .CreateMockClient(Logger.DefaultLogger, MockBehavior.Strict);
+            spannerClientMock
+                .SetupBatchCreateSessionsAsync()
+                .SetupExecuteStreamingSql();
 
             SpannerConnection connection = BuildSpannerConnection(spannerClientMock);
             var queryOptions = QueryOptions.Empty.WithOptimizerVersion(connOptimizerVersion);
+            queryOptions = queryOptions.WithOptimizerStatisticsPackage(connOptimizerStatisticsPackage);
             connection.QueryOptions = queryOptions;
 
             var command = connection.CreateSelectCommand("SELECT * FROM FOO");
             using (var reader = command.ExecuteReader())
             {
-                // Do nothing.
+                Assert.True(reader.HasRows);
```

Comment:
This bit is in the other PR I've just approved, so should disappear after merge and rebase, yes?

Reply:
Ah not quite. In #5292, I explicitly didn't modify all of these instances because I was already making the change in this PR. I only made changes to the emulator tests and not to the query options tests. These will stay in this PR (unless I should move this work to a separate PR?).